src/main/java/com/example/event_manager/exception/ExceptionController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.event_manager.exception;
+
+import java.util.NoSuchElementException;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.ServletWebRequest;
+import org.springframework.web.context.request.WebRequest;
+
+@ControllerAdvice
+public class ExceptionController {
+
+  @ExceptionHandler(value = NoSuchElementException.class)
```

Comment:
You should not catch java exception here rather than Your own exception.
In future there might be any other code which would throw this exception but You might want to not handle it in this way.